index.php

```diff
@@ -1 +1,28 @@
 <?php
+
+$db = new PDO('mysql:host=db; dbname=booker_winners', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('SELECT `ID`, `prize_year`, `author_name`, `book_name`, `author_nationality` FROM `booker_winners`;');
+$query->execute();
+$result = $query->fetchAll();
+
+?>
+
+<html>
+<body>
```

Comment:
Don't forget about the head section - we probably still want to include things like normalize and the responsive meta tag.